frontend/src/containers/ActivityInfo/ActivityInfo.js

```diff
@@ -1,36 +1,53 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
+import { Typography, Paper, Grid } from '@material-ui/core';
+import ActivityCard from '../../components/ActivityCard/ActivityCard';
+import './ActivityInfo.css';
+import githubLogo from './github-logo.png';
+import stackLogo from './stack-logo.png';
+import dockerLogo from './docker-logo.png';
 
 class ActivityInfo extends Component {
   render() {
     let { info } = this.props;
 
     return (
       <div className="activity-info">
-        <p>Username: @{ this.props.info.login }</p>
-        <div className="b-activity activity-info__github">
-          <p>Github</p>
-          { !info.github.error ? (
-            <ul>
-              <li>Loading: { this.props.info.github.isLoading ? '+' : '-'}</li>
-              <li>Name: { this.props.info.github.name || '(not set)'}</li>
-              <li>Languages: { info.github.languages && info.github.languages.length ?
-                info.github.languages.map((item, i) => <span key={i}>{item} </span>) :
-                'no languages'
-              }</li>
-            </ul>) : (
-            <p>{ info.github.error }</p>
-            )
-          }
-        </div>
-        <div className="b-activity activity-info__stackoverflow">
-          <p>Stackoverflow</p>
-          @TODO
-        </div>
-        <div className="b-activity activity-info__dockerhub">
-          <p>Dockerhub</p>
-          @TODO
-        </div>
+        <Paper className="activity-info__common" elevation={1}>
+          <Typography variant="h5" component="h3">
+            Common info
+          </Typography>
+          <Typography component="p">
+            <b>Username:</b> @{ this.props.info.login }
+          </Typography>
+        </Paper>
+
+        <Grid container spacing={16} alignItems="stretch" className="activity-cards-wrapper">
+          <Grid item xs={12} sm={4}>
+            <ActivityCard
+              title="Github Profile"
+              logo={githubLogo}
+              isLoading={info.github.isLoading}
+              name={info.github.name}
+              tags={info.github.languages}
+              error={info.github.error}
+            />
+          </Grid>
+          <Grid item xs={12} sm={4}>
```

Comment:
I just thought that we should hide activity cards for services which are not implemented yet or which analyses have failed.

Reply:
Not implemented - yes
But i think if we have error - we have to specify error reason(no profile, network error, etc) and show message in card

Reply:
Our goal is to build something that looks like a CV. Probably, it is not a good idea to show errors right there. Probably, some notification will be of help here.